Extension/src/main.ts

```diff
@@ -407,7 +425,8 @@ function rewriteManifest(): Promise<void> {
         "onCommand:C_Cpp.VcpkgClipboardInstallSuggested",
         "onCommand:C_Cpp.VcpkgClipboardOnlineHelpSuggested",
         "onDebug",
-        "workspaceContains:/.vscode/c_cpp_properties.json"
+        "workspaceContains:/.vscode/c_cpp_properties.json",
+        "onFileSystem:cpptools-schema"
```

Comment:
This is the fix. The other change of moving the register doesn't appear to affect anything, so it's more like a refactoring.